apps/api/src/routes/import-export.ts

```
@@ -0,0 +1,293 @@
+    }
+  },
+});
+
+// Initialize service lazily
+let importExportService: ImportExportService | null = null;
+
+// Helper function to get or create the service
+const getImportExportService = (): ImportExportService => {
+  if (!importExportService) {
+    if (!AppDataSource || !AppDataSource.isInitialized) {
+      throw new Error("Database not initialized");
```

ESLint

[error] Strings must use singlequote.

(quotes)